### ISSUE TITLE
qt5-base needs a sync to aur for openssl libs

### ISSUE THREAD

**Issue by xantares** [Martchus/PKGBUILDs#34]

**Comment by Martchus** [comment]
> export OPENSSL_LIBS -lssl -lcrypto, those are 1.1?

Referring to line?

```
export OPENSSL_LIBS='-L/usr/${_arch}/lib/openssl-1.0 -lssl -lcrypto'
```

See https://github.com/Martchus/PKGBUILDs/issues/32. The line has no effect (due to wrong quotes) but is not required anyways because OpenSSL is loaded at runtime. Since it makes no difference whether the line is present or not I didn't update the AUR.

BTW: In next update (to Qt 5.9.1) I'll change the config of the static variant to link against OpenSSL statically because it makes more sense for the static build to include everything in one executable. Here the line becomes relevant again.

For the dynamic variant I'd like to stick with the runtime loading approach. However, I still have to test whether this actually works because the patch in mingw-w64-openssl-1.0 to avoid symbol clash might be problematic. Have you already tested whether using dynamic variant with mingw-w64-openssl-1.0 works?

**Comment by xantares** [comment]
i'm using the dynamic variant, my app freezes on native windows, and I'm trying to see if it's related to qt5 alone or that opessl problem as my other dependencies require openssl 1.1.

Also I tried to link qt5-base to openssl 1.1 directly with this patch:
https://github.com/patch-exchange/openssl-1.1-transition/tree/master/qt5-qtbase
It provides a very good start but a few more symbols should be patched for 5.9.
